packages/neo-one-server-plugin/src/version.ts

```diff
@@ -1,2 +1,5 @@
+import * as fs from 'fs-extra';
+import path from 'path';
+
 // tslint:disable-next-line export-name
-export const VERSION = '1.0.0-preview.1';
+export const VERSION = JSON.parse(fs.readFileSync(path.resolve(__dirname, '..', 'package.json')).toString()).version;
```

Comment:
Use `fs.readFileSync(..., 'utf8')` and remove `toString`

Reply:
👍